components/autocomplete/src/autocomplete.spec.ts

```diff
@@ -1629,10 +1629,10 @@ class SimpleAutocomplete implements OnDestroy {
   openedSpy = jest.fn();
   closedSpy = jest.fn();
 
-  @ViewChild(DtAutocompleteTrigger, { static: false })
+  @ViewChild(DtAutocompleteTrigger)
   trigger: DtAutocompleteTrigger<any>;
-  @ViewChild(DtAutocomplete, { static: false }) panel: DtAutocomplete<any>;
-  @ViewChild(DtFormField, { static: false }) formField: DtFormField<any>;
+  @ViewChild(DtAutocomplete) panel: DtAutocomplete<any>;
+  @ViewChild(DtFormField) formField: DtFormField<any>;
   @ViewChildren(DtOption) options: QueryList<DtOption<any>>;
```

Comment:
Lots of public documentations missing. I know this is not part of the upgrade, just mentioning.

Reply:
Yea but I won't document this in this pr. I have not even written this code.